### PR TITLE
FF and Rewind added to GPIO_SETTINGS.ini

### PR DESCRIPTION
**Pull request by Caliandroid**
Added two new entries for Fast Forward and Rewind in the gpio_settings.ini  (disabled by default)

### CODE REVIEW

**Review by s-martin — CHANGES_REQUESTED** [review]
I would suggest to use pins not already used in the file, so users using this file directly don't end in a conflict.

**Comment by Caliandroid** [comment]
@s-martin : Good point! I've changed the pins to unused (7 & 8) + changed the double-usage pin 21 for RFID

File: components/gpio_control/example_configs/gpio_settings.ini [hunk 1]
```diff
@@ -83,10 +83,26 @@ pull_up: True
 hold_time: 0.3
 functionCall: functionCallPlayerPrev
 
+[FastForward]
+enabled: false
+Type:  Button
+Pin: 7
```
Comment (Caliandroid) [inline components/gpio_control/example_configs/gpio_settings.ini:89]: Changed to unused pin 7

File: components/gpio_control/example_configs/gpio_settings.ini [hunk 2]
```diff
@@ -83,10 +83,26 @@ pull_up: True
...
+pull_up: True
+hold_time: 0.3
+functionCall: functionCallPlayerSeekFwd
+
+[Rewind]
+enabled: false
+Type:  Button
+Pin: 8
```
Comment (Caliandroid) [inline components/gpio_control/example_configs/gpio_settings.ini:97]: Changed to unused pin 8

File: components/gpio_control/example_configs/gpio_settings.ini [hunk 3]
```diff
@@ -83,10 +83,26 @@ pull_up: True
...
+pull_up: True
+hold_time: 0.3
+functionCall: functionCallPlayerSeekBack
+
 [Halt]
 enabled: False
 Type:  Button
-Pin: 21
+Pin: 25
```
Comment (Caliandroid) [inline components/gpio_control/example_configs/gpio_settings.ini:105]: changed to resolve double-usage of the same pin as in usage for RFID

**Review by Caliandroid — COMMENTED** [review]
double usasge pin numbers issue resolved

**Review by s-martin — APPROVED** [review]

**Pull request merged**